### DISCUSSION TITLE
Error: [Reanimated] The native part of Reanimated doesn't seem to be initialized.

### DISCUSSION THREAD

**Discussion by SitaMelark** [software-mansion/react-native-reanimated#5023] [Q&A]
ERROR  Error: [Reanimated] The native part of Reanimated doesn't seem to be initialized. This could be caused by
- not rebuilding the app after installing or upgrading Reanimated
- trying to run Reanimated on an unsupported platform
- running in a brownfield app without manually initializing the native library, js engine: hermes
 LOG  Running "Gestorum" with {"rootTag":51}
 ERROR  Invariant Violation: "Gestorum" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes

Im trying to launch the app but i always get same issue. Tried to solve it but couldnt, but sometimes it gets solved by itself. This error usually happends to me when i create a new project, do the imports and i set my code, but sometimes whe i turn on the pc the next day or so, the issue is gone.

**Comment by piaskowyk** [comment]
Did you configure Reanimated Babel plugin? https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/#step-2-add-reanimateds-babel-plugin

**Reply by SitaMelark** [reply]
Already did that and i just found the solution, did a npm install and started working dunno why but thanks
